packages/features/bookings/lib/handleNewBooking.ts

```diff
@@ -668,20 +682,6 @@ async function handler(
   const seed = `${organizerUser.username}:${dayjs(reqBody.start).utc().format()}:${new Date().getTime()}`;
   const uid = translator.fromUUID(uuidv5(seed, uuidv5.URL));
 
-  let locationBodyString = location;
-  let defaultLocationUrl = undefined;
-  if (dynamicUserList.length > 1) {
-    users = users.sort((a, b) => {
-      const aIndex = (a.username && dynamicUserList.indexOf(a.username)) || 0;
-      const bIndex = (b.username && dynamicUserList.indexOf(b.username)) || 0;
-      return aIndex - bIndex;
-    });
```

Comment:
On line 639, we assigned the first entry in the `users` array as the organizer. Here we rearrange the `users` array so there's no guarantee that the first entry in the array will stay the same. I moved these lines of code earlier in the function so when we assign the organizer, the `users` array will not change anymore.